setup.py

```diff
@@ -13,7 +13,7 @@
     packages=find_packages(exclude=('example', 'tests')),
     install_requires=[
         'Django>=2.2',
-        'django_otp>=0.6.0,<0.99',
+        'django_otp>=0.6.0',
```

Comment:
Do you think it's safe to have no upper limit? django-otp does publish quite often and has been known to break things at random. Will that change post 1.0.0?